### DISCUSSION TITLE
Apply filters / blur to a specific part of the image

### DISCUSSION THREAD

**Discussion by puntiz** [hm21/pro_image_editor#243] [Q&A] [answered]
Hi,

Love the package. Is there currently anyway of applying various available filters to a specific area of the image rather than the whole image i.e., you select an area of the image and then apply blur or any other filter

**Comment by hm21** [comment, marked as answer]
Hi,

Currently, this isn’t possible because the image is simply wrapped inside the Flutter widget `ColorFiltered` to apply filters. To implement this feature, it may be necessary to overlay the same image as a background, use `CustomClipper` to crop it, and apply the desired filter there. Although this approach may seem straightforward, it’s somewhat complex, as it requires careful alignment with the transformations applied to the background image.
